### ISSUE TITLE
Luser comment: test recording upload

### ISSUE THREAD

**Issue by klausriede** [BioAcoustica/BioAcoustica#232]
Testing the upload function. I found (and entered some) of the metadata for the recording, however I did not see a function for uploading the wav/mp3 file...
Just wondering if this could go the other way round: first drop in a file (lookandfeel like youtube), and then bioacoustica extracts metadata from the RIFF file (which I will pre-edit for all files uploaded in the future)...just an idea

Issue on page http://bio.acousti.ca/content/test-recording-upload

**Comment by edwbaker** [comment]
You should be able to upload on the source tab:

![Screenshot 2019-06-24 at 15 09 08](https://user-images.githubusercontent.com/929684/60025713-2021fa00-9692-11e9-9359-042a29106dfd.png)

**Comment by edwbaker** [comment]
Good idea about RIFF, this was done previously for images within Scratchpads: https://bdj.pensoft.net/articles.php?id=973 - although would be good to check whether other Drupal modules may offer this already. I will make a new issue for this.

**Comment by klausriede** [comment]
no idea how to get there!
I managed to add metadata for a recording
http://bio.acousti.ca/content/crepitation-undetermined-acridinae-species-grasshopper#overlay-context=
and now I want to add the file - but how?
greetings Klaus


Klaus Riede Global REgister of Migratory Species www.groms.de<http://www.groms.de>


________________________________

**Comment by edwbaker** [comment]
It's on the same page as the metadata - just under a different tab, you need to click on the 'Source' tab.

![Screenshot 2019-06-25 at 13 47 58](https://user-images.githubusercontent.com/929684/60099646-fd561b00-974f-11e9-9f31-6ffe9b010ec1.png)

**Comment by klausriede** [comment]
ok, done. In the future I will start with an upload of the recording.
It seems once the metadata are there I cannot upload the soundfile in a second step.
And why do I have to fill out an URL? I added the inaturalist url where I entered the same observation.
This is a stereo recording, one channel filtered - can I switch the display?
Best wishes Klaus


Klaus Riede Global REgister of Migratory Species www.groms.de<http://www.groms.de>


________________________________

**Comment by edwbaker** [comment]
The URL is only used if the recording is already elsewhere on the web, it is not a required field.

**Comment by edwbaker** [comment]
Re: stereo waveforms see new issue: https://github.com/BioAcoustica/BioAcoustica/issues/234
